services/121-service/src/registration/services/registrations-bulk.service.ts

```diff
@@ -597,7 +597,7 @@ export class RegistrationsBulkService {
     await this.safaricomRequestScopedRepository
```

Comment:
we cannot just move queries in that new custom repo, because that is not scoped. Maybe get rid of the custom repo as it has only one simple method any more.

Reply:
see separate task 30447